src/modules/scraper/service/DepositsGapDetectionCron.ts

```diff
@@ -0,0 +1,173 @@
+import { Injectable, Logger } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import { LessThan, MoreThan, Repository } from "typeorm";
+
+import { EnhancedCron } from "../../../utils";
+import { AppConfig } from "../../configuration/configuration.service";
+import { Deposit } from "../../deposit/model/deposit.entity";
+import { MonitoringService } from "../../monitoring/service";
+import { DepositGapInterval, DepositGapService } from "./DepositGapService";
+
+export type DepositGapIntervalWithBlocks = DepositGapInterval & {
+  previousKnownDeposit?: { depositId: number; blockNumber: number };
+  nextKnownDeposit?: { depositId: number; blockNumber: number };
+};
+
+@Injectable()
+export class DepositsGapDetectionCron {
+  private logger = new Logger(DepositsGapDetectionCron.name);
+  private lock = false;
+
+  constructor(
+    private appConfig: AppConfig,
+    @InjectRepository(Deposit) private depositRepository: Repository<Deposit>,
+    private monitoringService: MonitoringService,
+    private depositGapService: DepositGapService,
+  ) {}
+
+  @EnhancedCron("0 5/10 * * * *")
```

Comment:
OOC - why is this crontab run `At minute 0 past every 10th hour from 5 through 23.`?

Reply:
it is running every ten minutes starting with minute 5